lib/ansible/template/__init__.py

```diff
@@ -255,6 +260,84 @@ def resolve_or_missing(self, key):
         return val
 
 
+class JinjaPluginIntercept(MutableMapping):
+    def __init__(self, delegatee, pluginloader, *args, **kwargs):
+        super(JinjaPluginIntercept, self).__init__(*args, **kwargs)
+        self._delegatee = delegatee
+        self._pluginloader = pluginloader
+
+        if self._pluginloader.class_name == 'FilterModule':
+            self._method_map_name = "filters"
```

Comment:
Can we please use the same quoting style?
```suggestion
            self._method_map_name = 'filters'
```